frontend/src/views/web/challenge/my-submission.html

```diff
@@ -78,6 +79,12 @@
                                     </thead>
                                     <tbody>
                                         <tr ng-repeat="key in challenge.submissionResult.results" class="result-val">
+                                            <td ng-if="challenge.isChallengeHost" class="center">
```

Comment:
Can we please move this checkbox near to the `is_public`. It'll be easy for the user.

Reply:
Done
![Screenshot from 2019-04-05 01-52-36](https://user-images.githubusercontent.com/3920286/55586000-8e1aed00-5745-11e9-969f-b0d05e6bf570.png)